modules/nw-nwop-viewing-config.adoc

```diff
@@ -0,0 +1,94 @@
+// Module filename: nw-nwop-viewing-config.adoc
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id='nw-nwop-viewing-config-{context}']
+= Viewing the Network Configuration in a Running Cluster
+
+
+
+.Prerequisites
+
+* Start a cluster and login using your credentials.
+
+.Procedure
+
+. Use the following command to view the `Network.config.openshift.io` CR.
++
+----
+$ oc get Network.config.openshift.io -o yaml
+----
++
+The cluster used in the example output below uses the default configuration values.
```

Comment:
not below.
> ! below adj
Do not use to indicate a relative location in a document, as in “the information below.” Use “the information
later in this section.”
Alternatively, you can use “following” to refer to a table, to art, or to a list. Example: “The following table
shows ...”

Maybe "The following output contains the default configuration values."